include/labwc.h

```diff
@@ -78,6 +78,9 @@ struct seat {
 	 */
 	struct cursor_context_saved pressed;
 
+	/* Cursor context of the last cursor motion */
+	struct cursor_context_saved hovered;
```

Comment:
Suggest `s/hovered/prev_cursor_context/` or something like that. When reading the code further down without the hint that it's a `struct cursor_context_saved` the term hovered isn't that intuitive.